internal/aws/ssm_client.go

```diff
@@ -22,18 +23,26 @@ func GetSSMDocumentName() (string, error) {
 	return name, nil
 }
 
-// SSMAPI interface for SSM operations
-type SSMAPI interface {
+// SSMClientInterface defines the interface for SSM operations we need
+type SSMClientInterface interface {
```

Comment:
add the comment again. We need get active session number and enforce a per MI active session number.